install/kubernetes/helm/istio/values.yaml

```diff
@@ -13,7 +13,7 @@ global:
   # Currently using nightly build, for testing
   tag: circleci-nightly
   proxy:
-    image: proxy
+    image: proxyv2
```

Comment:
I would simply replace proxy with proxyv2 and completely get rid of the old proxy. This way, we don't have to change the names anywhere. Are we offering upgrade path from 0.7.x directly to 1.0 ? or from 0.5.x to 1.0 ? if not, then we are better off going all in on proxy. Can be done in another PR

Reply:
Agreed, that's the plan. Including 'another PR' :-)

We do want to upgrade from 0.8 post transition ( i.e. with proxyv2) to 1.0 - and 0.8 will have proxyv2 injected.
So we'll likely keep proxyv2, and get rid of the old proxy images since we'll not support them anymore.